man/sysupdate.d.xml

```diff
@@ -595,17 +595,25 @@
 
         <para>If the source type is selected as <constant>url-file</constant> or
         <constant>url-tar</constant> this must be a HTTP/HTTPS URL. The URL is suffixed with
-        <filename>/SHA256SUMS</filename> to acquire the manifest file, with
-        <filename>/SHA256SUMS.gpg</filename> to acquire the detached signature file for it, and with the file
-        names listed in the manifest file in case an update is executed and a resource shall be
-        downloaded.</para>
+        the value assigned to the <varname>ManifestFile</varname> variable.</para>
```

Comment:
please suffix settings names with `=` in our docs and in log messages to indicate what they are